README.md

```diff
@@ -38,10 +10,10 @@ After you're finished please remove all the comments and instructions!
 
 # 📗 Table of Contents
 
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
+- [📖 About the Project]( this is a design replication task)
+  - [🛠 Built With](Html/css)
+    - [Tech Stack](html/css))
+    - [Key Features](hover)
   - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
```

Comment:
- [ ] Great job forbut I think you forget adding appropirate acknowledgment ,Therefore you can add it and add a ink to the design as well it would be great if you add it 🔥✨⭐💯🚀Like this ⬇⬇

Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).